build/integration.sh

```diff
@@ -26,7 +26,7 @@ printf "" # Refresh sudo credentials if necessary.
 function start {
   set +e  # We want to handle errors if cAdvisor crashes.
   echo ">> starting cAdvisor locally"
-  GORACE="halt_on_error=1" ./cadvisor --docker_env_metadata_whitelist=TEST_VAR --v=6 --logtostderr $CADVISOR_ARGS &> "$log_file"
+  GORACE="halt_on_error=1" ./cadvisor --disable_metrics="memory_numa,tcp,udp,advtcp,sched,process,hugetlb,referenced_memory,cpu_topology,resctrl" --docker_env_metadata_whitelist=TEST_VAR --v=6 --logtostderr $CADVISOR_ARGS &> "$log_file"
```

Comment:
I think it would be good to have a comment above this telling which metrics should NOT be disabled.

Reply:
Good idea :+1: